packages/hawtio/src/util/objects.ts

```diff
@@ -3,6 +3,14 @@ export function isObject(value: unknown): value is object {
   return value != null && (type === 'object' || type === 'function')
 }
 
+export function cloneObject<T>(value: T): T {
+  return JSON.parse(JSON.stringify(value))
+}
+
 export function isEmpty(obj: object): boolean {
   return Object.keys(obj).length === 0
 }
+
+export function isString(value: unknown) {
```

Comment:
let's declare return type as `value is string` to make our type-safe programming easier with it.